sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -174,7 +174,7 @@ case class CreateViewCommand(
       def verify(child: LogicalPlan) {
         child.collect {
           // Disallow creating permanent views based on temporary views.
-          case UnresolvedRelation(nameParts, _) if catalog.isTempView(nameParts) =>
+          case UnresolvedRelation(nameParts, _, _) if catalog.isTempView(nameParts) =>
```

Comment:
isStreaming = false only?

Reply:
We can create a temp view based on streaming relation, so it should be kept as a full match?

Reply:
+1